plugins/org.python.pydev.core/tests/org/python/pydev/core/TestDependent.java

```diff
@@ -194,6 +198,16 @@ public static boolean isUnix() {
                 System.err.println("PYTHON_EXE variable points to path that does NOT exist: " + PYTHON_EXE);
             }
 
+            if (CONDA_PYTHON_ENV != null) {
+                if (!new File(CONDA_PYTHON_ENV).exists()) {
+                    System.err.println(
```

Comment:
Instead of `System.err`, it should `Log` it.

Reply:
We should update the entire file to log errors instead of calling `System.err`?

Reply:
My bad, in TestDependent using System.err is expected.